demo/src/screens/MainScreen.js

```diff
@@ -174,17 +237,74 @@ class MainScreen extends Component {
         hideUnderline
         floatingPlaceholder={false}
         text70
-        trailingAccessory={
+        leadingAccessory={
           filterText ? (
-            <Button link iconSource={Assets.icons.demo.close} $iconDefault onPress={this.clearSearch}/>
+            <Button link marginR-5 iconSource={Assets.icons.demo.close} $iconDefault onPress={this.clearSearch}/>
```

Comment:
Let's have the cancel (x) button of the search input on the right side (using `trailingAccessory` as it was before)